### ISSUE TITLE
Create "Grants Pending" Page - Design

### ISSUE THREAD

**Issue by captnseagraves** [gitcoinco/web#1904]
### User Story

As a Grants user, I want a screen to let me know when my transaction is pending on-chain so I know that I have submitted my grant but it has not been successfully written yet. 

### Why Is this Needed

The "Grants Pending" page is needed to inform the user of the state of their transaction and to let them know that their grant is not active yet. 

### Description

*Type*: Feature
### Definition of Done

- [ ] Replicate the "Pending" page for bounties with Grants Copy
- [ ] Page has standard Gitcoin header and footer
- [ ] Design must be on brand, see creative repo for brand guide

### Additional Information

To be reviewed with @PixelantDesign @mbeacom @captnseagraves

Existing assets can be found at this PR: https://github.com/gitcoinco/web/pull/674
More information about the Dev Grants project can be found here: https://github.com/gitcoinco/web/issues/1469

Please allow the core team a few rounds of revisions.

**Comment by captnseagraves** [comment]
Assigning @justpixel

**Comment by owocki** [comment]
note: we *could* reuse the bounty pending page, so i think this is lower priority than the others

**Comment by PixelantDesign** [comment]
I agree on reusing the bounty pending page here.

**Comment by captnseagraves** [comment]
Sounds like we are just to replicate and change a little bit of copy

**Comment by thelostone-mc** [comment]
We'll be porting over the waiting state as is from bounty.
How it would work -> Once a grant is created , we display the waiting state until the txn has been confirmed 

@SaptakS has got a PR out for this. Only thing left is to handle for `web3js` version `1.0.0`

**Comment by owocki** [comment]
> We'll be porting over the waiting state as is from bounty.

+1 for anything that refactors `bounty_details.js` to be smaller :)

**Comment by thelostone-mc** [comment]
@captnseagraves yeah we'll add it in there.
@SaptakS would it make sense to spin up a new ticket and handle it all in one ticket ?

**Comment by captnseagraves** [comment]
@thelostone-mc @SaptakS We can handle it all here. I'm happy to implement today as well, if you haven't begun to move on it.

**Comment by SaptakS** [comment]
@captnseagraves these page is already implemented and there in the grants page. Can you please check. There is a waiting state page after creation of grants.

**Comment by captnseagraves** [comment]
@SaptakS Yes, the waiting state is done for grants/new. Thank you! There are two other points in the application where we send transactions to the blockchain. In the "Fund Grant" view and the "Cancel Subscription" view. These also need a waiting state.

**Comment by SaptakS** [comment]
@captnseagraves We can put the same one right? I will put them there too. :)

**Comment by captnseagraves** [comment]
@SaptakS Yes! Same one should be good.
